rosys/actors/lizard.py

```diff
@@ -33,3 +41,12 @@ async def ensure_responsiveness(self):
         elif dt > 0.1:
             self.log.warning(f'esp serial communication is slow ({dt:.2f} s since last step)')
         self.last_step = self.world.time
+        self.responsiveness_stats.append(dt * 100)
+
+    def limit_stat_values(self):
```

Comment:
can be avoided using [`collections.deque`](https://docs.python.org/2/library/collections.html#collections.deque)